lib/eventboss/queue.rb

```diff
@@ -23,6 +36,15 @@ def url
       @url ||= client.get_queue_url(queue_name: name).queue_url
     end
 
+    def arn
+      [
+        'arn:aws:sqs',
+        Eventboss.configuration.eventboss_region,
+        Eventboss.configuration.eventboss_account_id,
```

Comment:
`configuration` is set in the initializer but not used. Maybe we should remove from there or use it, wdyt?

Reply:
Done.